charts/hivemq-platform/values.schema.json

```diff
@@ -518,6 +516,59 @@
           "type" : "boolean"
         }
       }
+    },
+    "migration" : {
+      "type" : "object",
+      "properties" : {
+        "statefulSet" : {
+          "type" : "boolean"
+        }
+      }
+    }
+  },
+  "allOf": [
```

Comment:
I didn't know you could have such logic inside a JSON schema. This is impressive!